CHANGES

```diff
@@ -30,6 +30,10 @@ Pint Changelog
 - Fix parsing of units string with same canonalized name (Issue #1441 & #1142)
 - The pint-pandas example notebook has been moved to the pint-pandas package.
 
+### New Units
+- `sverdup`
```

Comment:
```suggestion
- `sverdrup`
```
Should this also include the PR numbers?

Reply:
Will do